Misc/NEWS.d/next/Library/2021-09-11-13-45-47.bpo-45172.T0LUgT.rst

```diff
@@ -0,0 +1 @@
+Adds CAN protocol constants for NetBSD to :mod:`socket`.
```

Comment:
The NEWS entry might be more specific: specify CAN_RAW_ERR_FILTER.

Reply:
I agree with Victor; explicit is better than implicit.